purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/properties/Padding.kt

```diff
@@ -0,0 +1,31 @@
+package com.revenuecat.purchases.paywalls.components.properties
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Contains padding values for 4 axes, in dp.
+ */
+@Serializable
+internal data class Padding(
```

Comment:
I think all these internal should have `@JvmSynthetic` to prevent exposing them into Java

Reply:
Ugh you're right! It gets a red underline in Java, but it compiles fine. 😓 

However, `@JvmSynthetic` cannot be applied to classes or constructors. Not sure if there's anything we can do. (Applying to the file doesn't work.)

```kotlin
@Target(AnnotationTarget.FILE, AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.FIELD)
@Retention(AnnotationRetention.SOURCE)
public actual annotation class JvmSynthetic
```

Reply:
Will merge for now, as the rest of the stack added way more `internal` types. So if we come up with any solution, I'll apply it to the top of the stack.